unaligned/align/cupid

```diff
@@ -4,4 +4,6 @@ The Cupid chooses a player to be their lover, and forms an inseparable team with
 __Details__
 During the first night, the Cupid must choose a player to become their lover. If the Cupid does not choose in time, they will fall in love with a random player. The lover is added to the Cupid’s secret channel.
 The Cupid and their lover form an inseparable team; they will die if the other player dies. Choosing a lover is an immediate ability.
-By default the Cupid wins with the townsfolk team. If the lover is on a different team, they will only win when both win conditions are fulfilled. For example, if the couple is Cupid and a werewolf, everyone else must die. Another example is when the couple is Undead and a Vampire, then the normal Vampire win condition must be reached. Or, if they’re both townsfolk, they belong to the townsfolk team and can win together with them. The Cupid’s love overrides any other mechanic that may cause a player to change teams for any reason; the Cupid is always on the same team as their lover. After choosing a lover, unless they are also unaligned, the Cupid is no longer unaligned.
+By default the Cupid wins with the townsfolk team. If the lover is on a different team, they will only win when both win conditions are fulfilled. For example, if the couple is Cupid and a werewolf, everyone else must die. If the lover is townsfolk, they belong to the townsfolk team and can win together with them. 
```

Comment:
should the cupid be default town-aligned? seems to punish wolves.

also, even if so, the "default" being town now only applies to town and wolves, so is no longer really default. would suggest phrasing change.

Reply:
The default still applies to other solo teams (Hell has an exception via Demonic Cupid - Undead Cupid is different from it as Undead Cupid's lover can be on any team)

Reply:
When a Cupid targets a wolf it basically turns it into a white werewolf without the extra kill and with a liability that it can't tell wolves about so it's really a nerfed www

Reply:
I'm not sure what alternatives there are? I guess just that the Cupid dies if they select a wolf?

Reply:
I think Cupid could possibly win with the alignment of whoever it targets, currently it's pretty much a loss for the Cupid if it targets a wolf or a solo since it has to outlive town, wolves and solo and as soon as wolves or solo catch on that Cupid targeted one of them then the they'll just kill Cupid, I don't know why Cupid as an Unaligned align has a base alignment if it's forced to target Night 1

Reply:
It's because Cupid originally is a Townsfolk role and it makes sense to be like this as we already have LA. Since it has a base alignment it's way less volatile.